code/modules/crafting/table.dm

```diff
@@ -1,7 +1,9 @@
-#define TABLECRAFT_MAX_ITEMS 30
 
 /obj/structure/table
 	var/list/table_contents = list()
+	var/viewing_category = 1 //typical powergamer starting on the Weapons tab
+	var/list/categories = list(CAT_WEAPON,CAT_AMMO,CAT_ROBOT,CAT_FOOD,CAT_MISC)
```

Comment:
If you arent changing it from table to table or anywhere really maybe make it a global var
I dont think each table needs a separate list for absolutely the same unchanging thing